.gitlab-ci.yml

```diff
@@ -2658,7 +2658,9 @@ send_pkg_size-a7:
     # Build testing stage if provided
     - test "$TESTING_ARG" && docker build --file $BUILD_CONTEXT/$ARCH/Dockerfile $TESTING_ARG $BUILD_CONTEXT
     # Build release stage and push to ECR
-    - docker build $BUILD_ARG --file $BUILD_CONTEXT/$ARCH/Dockerfile --pull --tag $TARGET_TAG $BUILD_CONTEXT
+    - docker build $BUILD_ARG --file $BUILD_CONTEXT/$ARCH/Dockerfile --pull --tag ${TARGET_TAG}-unsquashed $BUILD_CONTEXT
+    - pip install docker-squash
```

Comment:
I think the trick when experimental is not enabled is to use `docker export | docker import`

Reply:
Yeah, but doing this looses all the image metadata like `CMD`, `ENV`, etc. It creates a new mono-layer image with the same filesystem but all `CMD`, `ENV`, `VOLUMES`, etc. are lost.
And we need at least `CMD` to target S6 and some environment variables.

Reply:
Ah okay, my bad then. The only other option I see is to use `docker buildx build --squash` which requires experimental on client side (not sure if it also requires experimental on daemon side, not documented at least).

Reply:
Thanks for the tip @vboulineau !
I’ve switched to `docker buildx build --squash`. It works and it doesn’t require any 3rd party tool.

Reply:
`docker buildx build --squash` isn’t implemented:
https://github.com/docker/buildx/blob/master/commands/build.go#L76